wordpress-mobile/WordPress-iOS
# Stats Revamp: Insights Management UI tweaks

After chatting with @annchichi, this PR includes a few UI tweaks for the Insights Management screen to improve the UX a little.

| Before | After |
|---|---|
| ![Simulator Screen Shot - iPhone 13 Pro - 2022-05-26 at 14 34 45](https://user-images.githubusercontent.com/4780/170498407-14ad3bed-fcf7-4404-a774-41dd309f7ccb.png) | ![Simulator Screen Shot - iPhone 13 Pro - 2022-05-26 at 14 13 50](https://user-images.githubusercontent.com/4780/170496722-61100c3b-44ee-492e-b50c-ecff4839976e.png) |

**To test**

* With the two new stats feature flags disabled, build and run and navigate to Stats > Insights, and then tap the settings icon in the top right
* Ensure the screen still looks as it does in the shipping version of the app, and you can tap a single row to dismiss the screen and add that card to the Insights view

* Enable the two new Stats feature flags and rebuild and run
* Navigate back to Insights management settings
* Ensure that the view looks as in the screenshot above
* Try tapping rows to add and remove cards, and use the reorder controls
* Save your changes and ensure they're reflected in the main Insights screen

## Regression Notes

1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
